git-ghpr/bin/git-ghpr

```diff
@@ -27,6 +29,17 @@ EOF
     exit 1
 }
 
+github-list()
+{
+    local org repo
+    org=$(git remote get-url upstream | cut -d: -f2 | cut -d/ -f1)
+    repo=$(basename $(git remote get-url upstream | cut -d: -f2 | cut -d/ -f2) .git)
```

Comment:
This URL parsing only works for the legacy scp-like form of ssh "URL"s.
Proper URLs must be supported. Legacy "URL"s should be supported.